### ISSUE TITLE
Inkcyclopedia search rankings

### ISSUE THREAD

**Issue by terminalnode** [terminalnode/mrfreeze#7]
It would be cool to have a ranking of how many times a given ink has been looked up on the inkcyclopedia.

**Comment by terminalnode** [comment]
Progress on this currently blocked by: https://github.com/terminalnode/mrfreeze/issues/12

**Comment by terminalnode** [comment]
Probably still possible with the new API-based search. We'll simply have to store stats based on the full name (and possibly key) to see which inks get the most hits. Could still be a fun project I think, albeit low priority.

**Comment by terminalnode** [comment]
The main server which is using the bot, The Penposium, has chosen to deactivate the ink search functionality. As such there's no point in developing this features at this time.

Closing this. Might reopen if the ink search gets enabled in The Penposium again.
